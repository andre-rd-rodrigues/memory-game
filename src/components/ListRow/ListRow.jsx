import React from "react";
import styles from "./listrow.module.scss";

const ListRow = ({ label, content }) => {
  return (
    <div className={styles.listRow}>
      <p>{label}</p>
      <p>{content}</p>
    </div>
  );
};

export default ListRow;
