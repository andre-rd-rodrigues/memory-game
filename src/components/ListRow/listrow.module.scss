@import "styles/mixins";

.listRow {
  display: flex;
  justify-content: space-between;
  align-items: center;
  background-color: $secondary-light;
  padding: 0.7rem 1.3rem;
  margin-bottom: 1rem;
  border-radius: 0.4rem;
  box-shadow: 0 1px 2px rgba(38, 41, 44, 0.08);
  & p {
    margin-bottom: 0;
    font-size: 0.9rem;
    color: $primary-light;
    &:last-child {
      color: $primary;
      font-weight: bold;
      font-size: 1.3rem;
    }
  }
}
