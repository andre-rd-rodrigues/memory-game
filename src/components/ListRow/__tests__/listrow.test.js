import { shallow } from "enzyme";
import ListRow from "../ListRow";

describe("List row", () => {
  let props;
  props = {
    label: "Moves Taken",
    content: "10"
  };

  it("should render correctly", () => {
    const wrapper = shallow(<ListRow {...props} />);

    expect(wrapper).toMatchSnapshot();
  });
});
