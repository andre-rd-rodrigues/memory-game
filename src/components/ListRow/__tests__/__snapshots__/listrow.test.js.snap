// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List row should render correctly 1`] = `
<div
  className="listRow"
>
  <p>
    Moves Taken
  </p>
  <p>
    10
  </p>
</div>
`;
